### PR TITLE
fixup! [IMP] base_automation: simpler edition workflow

### PR DESCRIPTION
**Pull request by bouvyd**

### CODE REVIEW

File: addons/mail/models/ir_actions_server.py [hunk 1]
```diff
@@ -69,20 +68,24 @@ class ServerActions(models.Model):
         compute='_compute_activity_info', readonly=False, store=True)
     activity_user_type = fields.Selection(
         [('specific', 'Specific User'),
-         ('generic', 'Generic User From Record')],
+         ('generic', 'Dynamic User (based on record)')],
+         string='User Type',
         compute='_compute_activity_info', readonly=False, store=True,
         help="Use 'Specific User' to always assign the same user on the next activity. Use 'Generic User From Record' to specify the field name of the user to choose on the record.")
```
Comment (brboi) [inline addons/mail/models/ir_actions_server.py]: help should be adapted

File: odoo/addons/base/models/ir_actions.py [hunk 2]
```diff
@@ -496,28 +496,30 @@ def _default_value(self):
                                  string='Child Actions', help='Child server actions that will be executed. Note that the last return returned action value will be used as global return value.')
     # Create
     crud_model_id = fields.Many2one(
-        'ir.model', string='Target Model',
+        'ir.model', string='Record to Create',
         compute='_compute_crud_model_id', readonly=False, store=True,
-        help="Model for record creation. Set this field only to specify a different model than the base model.")
+        help="Specify which kind of record should be created. Set this field only to specify a different model than the base model.")
```
Comment (brboi) [inline odoo/addons/base/models/ir_actions.py:501]:
```suggestion
        help="Specify which kind of record should be created.")
```

`crud_model_id` is required when `state == 'object_create'` and invisible if `state != 'object_create'`

maybe we should rename the field `create_model_id` or smtg similar ?

Comment (bouvyd) [reply odoo/addons/base/models/ir_actions.py:501]: you can if you want :D

File: addons/mail/models/ir_actions_server.py (hunk 1 above)
Comment (bouvyd) [reply addons/mail/models/ir_actions_server.py]: yes

**Comment by bouvyd** [comment]
i'm also reconsidering the SMS wording for the 'send sms' methods/helper as what it says is still not correct, and i have my suspicisions for mail as well, but i'll commit directly to your branch i think

**Comment by bouvyd** [comment]
@brboi i've pushed all my changes, including the one that resolves the 'help' issue with regards to the label

**Pull request merged**